Formula/pyside.rb

```diff
@@ -58,7 +59,11 @@ def install
               "in_build = Path(\"@CMAKE_BINARY_DIR@\") in location.parents",
               "in_build = Path(\"@CMAKE_BINARY_DIR@\").resolve() in location.parents"
 
+    # Install python scripts into pkgshare rather than bin
+    inreplace "sources/pyside-tools/CMakeLists.txt", "DESTINATION bin", "DESTINATION #{pkgshare}"
```

Comment:
Alternatively, could manually move everything, which was how this was previously handled (`metaobjectdump.py`, `project.py`). New release has added more scripts causing auditing failure:
```
  pyside:
    * Non-executables were installed to "/opt/homebrew/opt/pyside/bin".
      The offending files are:
        /opt/homebrew/opt/pyside/bin/deploy.py
        /opt/homebrew/opt/pyside/bin/deploy
        /opt/homebrew/opt/pyside/bin/qtpy2cpp_lib
        /opt/homebrew/opt/pyside/bin/qtpy2cpp.py
        /opt/homebrew/opt/pyside/bin/qml.py
    * Files were found with references to the Homebrew shims directory.
      The offending files are:
        bin/shiboken6
  Error: 2 problems in 1 formula detected
```